### ISSUE TITLE
Introduce backups in Libero deployments

### ISSUE THREAD

**Issue by giorgiosironi** [libero/publisher#365]
Definition of Done:

**Comment by thewilkybarkid** [comment]
Do we actually need this? There is no 'real' Libero deployment...

**Comment by giorgiosironi** [comment]
No, we probably never need it; I am collecting these issues to write an ADR on what the Libero deployments are, which I suspect should be demonstration of features plus providing a working model for a service provider to fork and build on. If this is then widely accepted, this and other issues will likely be dropped as it's not needed for demonstration, plus every service provider would have its own way of doing it.
